src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import { Route } from 'react-router-dom';
 
 const App = () => (
   <div>
     <Navigation />
+      <Route exact path="/" component={Home} />
```

Comment:
Can you explain briefly . what exactly is `exact` doing here? What happens if we remove it?

Reply:
exact ensures the "Home" component specific to only path "/". Otherwise, Home component could be called while path "/about" or path "/contact" is called, since they each have the / in their paths.